### ISSUE TITLE
Remove Ikea Trolley From the Playroom

### ISSUE THREAD

**Issue by Stephanymtr** [bmhco/shared#19]
### **Task**
**Remove Ikea Trolley From the Playroom**
The RÅSKOG Trolley is currently being used to store art and craft supplies. A has gotten used to it, but it’s not the safest piece of furniture for a children’s space. It’s easy (and tempting) to climb and even easier to fall off of it. It’s not secured to the wall, and it contains wheels. I believe we are better off by permanently removing it out of the room and temporarily storing supplies with the toys until the new shelves are ready. What do you think ? I'm happy to do it right away. 

![IMG-4247](https://user-images.githubusercontent.com/91985721/213410456-059708eb-758c-4ff1-b179-59b96322558f.jpg)

**Comment by nelsonic** [comment]
Where would it be placed? is it not easier to remove the wheels so that there's less risk of it toppling over? 
Or somehow fixing it to the wall with some brackets? 💭

**Comment by Stephanymtr** [comment]
Yes, removing the wheels will prevent it from moving and securing the trolley to the wall will prevent it from falling onto the little ones. However, the trolley shelves don't allow enough room for displaying arts and crafts activities separately. It is being used for storing most of the supplies.

**Comment by nelsonic** [comment]
Ok. So would the trolley be retired from use or repurposed for something else? 💭

**Comment by Stephanymtr** [comment]
I'm sure the trolley would be useful upstairs... maybe to store bathroom products or drying rack/ironing accessories. This is just a suggestion. If you and/or @iteles feel we should keep it in the playroom, securing it to the wall will be enough.
